flink-libraries/flink-table/src/test/scala/org/apache/flink/api/scala/batch/table/CalcITCase.scala

```diff
@@ -109,36 +110,6 @@ class CalcITCase(
     TestBaseUtils.compareResultAsText(results.asJava, expected)
   }
 
-  @Test(expected = classOf[ValidationException])
```

Comment:
The test method `testAliasStarException()` should be moved to `CalcValidationTest` and be split up into individual methods.